rasa_nlu/featurizers/regex_featurizer.py

```diff
@@ -67,18 +67,26 @@ def features_for_patterns(self, message):
 
         Given a sentence, returns a vector of {1,0} values indicating which
         regexes did match. Furthermore, if the
-        message is tokenized, the function will mark the matching regex on
-        the tokens that are part of the match."""
+        message is tokenized, the function will mark all tokens with a dict
+        relating the name of the regex to whether it was matched."""
 
         found = []
         for i, exp in enumerate(self.known_patterns):
             match = re.search(exp["pattern"], message.text)
             if match is not None:
```

Comment:
let's move this if/else into the loop on L77 (because right now this loop is duplicated in L86)

Reply:
ok, I fixed this as well, although I had to make an additional small change.  Please take a look at the new commit when you get a second.